src/components/LanguageSwitch/LanguageSwitch.module.css

```diff
@@ -63,9 +65,13 @@
   display: inline-block;
 }
 
+.LanguageSwitchListItem:nth-child(1) {
+  margin-top: 10px;
+}
```

Comment:
This should be deleted. Instead, change the `top` value in the `.LanguageSwitchList` class to `30px`.